__tests__/utils/date.spec.ts

```diff
@@ -40,6 +41,8 @@ describe('date', () => {
   });
 
   test('formatDiff', () => {
+    registerBuiltInLocales();
```

Comment:
the test relies on built-in locales so required modifications

Reply:
just register twice.

I think can move formatDiff into format.ts to remove the dependence of locales.ts, your opinions?